moduleconfig/src/main/java/org/jboss/pnc/common/json/moduleconfig/OpenshiftEnvironmentDriverModuleConfig.java

```diff
@@ -92,9 +98,30 @@ public OpenshiftEnvironmentDriverModuleConfig(
         this.containerPort = containerPort;
         this.keepBuildAgentInstance = keepBuildAgentInstance != null ? keepBuildAgentInstance : false;
         this.exposeBuildAgentOnPublicUrl = exposeBuildAgentOnPublicUrl != null ? exposeBuildAgentOnPublicUrl : false;
-        this.creationPodRetry = creationPodRetry;
         this.builderPodMemory = builderPodMemory == null ? DEFAULT_BUILDER_POD_MEMORY : builderPodMemory;
 
+        this.creationPodRetry = DEFAULT_CREATION_POD_RETRY;
+        if (creationPodRetry != null) {
+            try {
+                this.creationPodRetry = Integer.parseInt(creationPodRetry);
+            } catch (NumberFormatException e) {
+                log.error(
+                        "Couldn't parse the value of creation pod retry from the configuration. Using default ({} retries)",
+                        DEFAULT_CREATION_POD_RETRY);
+            }
+        }
+
+        this.pollingMonitorTimeout = PollingMonitor.DEFAULT_TIMEOUT;
+        if (pollingMonitorTimeout != null) {
```

Comment:
Since the value is assigned in the previous line from the constant, what is the case for null ?

Reply:
pollingMonitorTimeout comes from the config file mapping and is passed as parameter, it's not the same variable